src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/JointCNVSegmentation.java

```diff
@@ -0,0 +1,172 @@
+package org.broadinstitute.hellbender.tools.walkers.sv;
+
+import com.google.common.collect.Lists;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.variant.variantcontext.*;
+import htsjdk.variant.variantcontext.writer.VariantContextWriter;
+import htsjdk.variant.vcf.VCFConstants;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import htsjdk.variant.vcf.VCFStandardHeaderLines;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.StructuralVariantDiscoveryProgramGroup;
+import org.broadinstitute.hellbender.engine.MultiVariantWalkerGroupedOnStart;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.gcnv.GermlineCNVSegmentVariantComposer;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFConstants;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFHeaderLines;
+import org.broadinstitute.hellbender.tools.sv.SVCallRecord;
+import org.broadinstitute.hellbender.tools.sv.SVClusterEngine;
+import org.broadinstitute.hellbender.tools.sv.SVCallRecordWithEvidence;
+import org.broadinstitute.hellbender.tools.sv.SVDepthOnlyCallDefragmenter;
+import org.broadinstitute.hellbender.utils.IntervalUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.genotyper.IndexedSampleList;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFHeaderLines;
+import org.broadinstitute.hellbender.utils.variant.HomoSapiensConstants;
+
+import java.io.File;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@BetaFeature
+@CommandLineProgramProperties(
+        summary = "Gathers single-sample segmented gCNV VCFs, harmonizes breakpoints, and outputs a cohort VCF with genotypes.",
+        oneLineSummary = "Combined single-sample segmented gCNV VCFs.",
+        programGroup = StructuralVariantDiscoveryProgramGroup.class
+)
+public class JointCNVSegmentation extends MultiVariantWalkerGroupedOnStart {
+
+    private SortedSet<String> samples;
+    private VariantContextWriter vcfWriter;
+    private SAMSequenceDictionary dictionary;
+    private SVDepthOnlyCallDefragmenter defragmenter;
+    private SVClusterEngine clusterEngine;
+
+    private String currentContig;
+
+    public static final String MIN_QUALITY_LONG_NAME = "minimum-qs-score";
+
+    @Argument(fullName = MIN_QUALITY_LONG_NAME, doc = "Minimum QS score to combine a variant segment")
+    private int minQS = 20;
```

Comment:
That's still what Isaac has in the latest version.  Higher would be more efficient (i.e. merging fewer calls), but this is safer in case someone reduces the QS filter threshold downstream.